RUFAS/weather.py

```diff
@@ -80,6 +91,8 @@ def __init__(self, weather_file: dict, time: RufasTime):
 
         self.mean_annual_temperature = self._calculate_average_annual_temperature(weather_file["avg"])
 
+        self.set_LINEST_temperature_factors()
```

Comment:
```suggestion
        self.set_linest_temperature_factors()
```

not sure we should violate function case conventions for this.